docker/docker-compose-dev-essentials.yaml

```diff
@@ -8,8 +8,8 @@ services:
     ports:
       - "5432:5432"
     volumes:
-      - postgres_data:/var/lib/postgresql/data/
-      - ../backend/init.sql:/docker-entrypoint-initdb.d/init.sql
+      - postgres_data_oss:/var/lib/postgresql/data/
```

Comment:
@chandrasekharan-zipstack no need add `oss` suffix anywhere in code.

Reply:
@jaseemjaskp thanks for pointing out, this was added by mistake. I had made this change while testing. Will remove it